Documentation/Configuration.md

```diff
@@ -0,0 +1,23 @@
+# Configuring WheelMUD
+
+## Server Data
+When building and running the server for the first time, a bunch of files are copied over to the server's "program data" location where it can persist and evolve (such as when new characters are saved to the database).
```

Comment:
Currently it's by design, but I'm not opposed to it changing.  It is kind of annoying to track down, even though it's technically where Windows apps are _supposed_ to be putting such persisting data.